packages/api/src/controllers/v2/reviews/mod.rs

```diff
@@ -0,0 +1,197 @@
+#[allow(unused)]
+use by_axum::axum::{
+    extract::{Path, Query, State},
+    routing::post,
+    Json,
+};
+use by_axum::{auth::Authorization, axum::Extension};
+use models::{
+    review::{
+        Review, ReviewAction, ReviewByIdAction, ReviewCreateRequest, ReviewGetResponse,
+        ReviewParam, ReviewQuery, ReviewQueryActionType, ReviewRepository,
+        ReviewRepositoryUpdateRequest, ReviewSummary, ReviewUpdateRequest,
+    },
+    *,
+};
+use sqlx::postgres::PgRow;
+
+#[derive(Clone, Debug)]
+pub struct ReviewControllerV2 {
```

Comment:
```suggestion
pub struct ReviewController {
```

Module structure already express version

Reply:
I updated the naming.